docker-compose.besu-nimbus2.yml

```diff
@@ -0,0 +1,96 @@
+version: "3.8"
+
+services:
+  besu:
+    image: hyperledger/besu:$BESU_VERSION
+    container_name: besu
+    volumes:
+      - $EXECUTION_DATA_VOLUME:/execution_data
+      - $CONFIGS_VOLUME:/configs
+    restart: unless-stopped
+    stop_signal: SIGINT
+    stop_grace_period: 2m
+    command: >
+      --data-path=/execution_data
+      --genesis-file=/configs/genesis.json
+      --min-gas-price=1000000000
```

Comment:
Please change to default one: https://github.com/lukso-network/network-configs/blob/main/mainnet/besu/besu.toml